### ISSUE TITLE
Request: From scratch tutorial

### ISSUE THREAD

**Issue by emilianbold** [owntracks/owntracks#28]
The booklet is quite dense and each sub-project seems to explain many options and tricks but I think as a product OwnTracks documentation doesn't provide a clear 'happy path' for users starting from scratch.

I suggest a step by step tutorial that assumes no existing infrastructure except having mobile phones.

So:

1. The user has a group of friends, each with iOS / Android phones. This step is easy: just install the app from the respective store.
2. The user needs to have a backend. This step is hard.
2.1 The user could use VPSs from the cloud or local machines.
2.1 What/how to install?
2.2 How to configure the backend/mobile in order to give access to each friend in the group? How to add/remove friends?
3. Enjoy seeing the friend's location.

Ideally the product should also think about how backend updates should happen but this is not mandatory.

**Comment by stevenroose** [comment]
I agree with this.. I think the documentation is kinda a chaos and it doesn't even explain well what you actually are supposed to do. When you "follow" the steps, it says you have "2 options" and then when going the option mqtt, it stops after it tells you how to install the mqtt server. It doesn't tell you anything about what to do owntracks-wise afterwards...

**Comment by IcyAlmond** [comment]
this is very needed, there are unexplained things along each step and the docker-compose.yml files don't give a working installation.

it's not clear if I need to use `recorder` if I've setup a mosquitto broker. And as stevenroose said, it's not clear what to do after installing the mqtt broker. The mobile app's "connection" portion does not have clear documentation. so I don't know how to connect to the mqtt server properly.
